resources/lang/zh/accounts.php

```diff
@@ -20,7 +20,7 @@
 
 return [
     'edit' => [
-        'title' => '账户设置',
+        'title' => '<strong>账户</strong>设置',
```

Comment:
in chinese, it means "Account Settings", not "All Settings". Bold can make users clearly know it.